pkgs/top-level/all-packages.nix

```diff
@@ -35928,7 +35928,9 @@ with pkgs;
 
   bcompare = libsForQt5.callPackage ../applications/version-management/bcompare {};
 
-  pentablet-driver = libsForQt5.callPackage ../misc/drivers/pentablet-driver { };
+  xp-pen-deco-01-v2-driver = libsForQt5.xp-pen-deco-01-v2-driver;
+
+  xp-pen-g430-driver = libsForQt5.callPackage ../os-specific/linux/pentablet-drivers/xp-pen-g430-driver { };
```

Comment:
Would you mind moving this to `qt5-packages.nix` and use `libsForQt5.xp-pen-g430-driver` here as well? Not really a blocker for this PR, but would be nice for consistency.